db/migrate/20240422172820_add_position_to_marketplace_tag.rb

```diff
@@ -0,0 +1,9 @@
+class AddPositionToMarketplaceTag < ActiveRecord::Migration[7.1]
+  def change
+    add_column :marketplace_tags, :position, :integer
+
+    # Marketplace::Tag.order(:updated_at).each.with_index(1) do |tag, index|
+    #   tag.update_column :position, index
+    # end
```

Comment:
I don't think this is necessary but we could.

Reply:
This makes me think that a `Group` wants to be it's own table, rather than a flag on `marketplace_tags`, but also I don't want to eat that complexity right now. I think ordering by alpha is fine.

Reply:
_:nod:_ I started going down the path of writing a `Group` model and I just couldn't come up with a strong enough justification at this time since the goal of this changeset feels achievable without it.